<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VotingApp</class>
 <widget class="QWidget" name="VotingApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voting App</string>
  </property>
  <widget class="QComboBox" name="candidateDropdown">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>John</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jane</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="voteButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Vote</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
